src/js/script.js

```diff
@@ -32,8 +32,21 @@
       },
     },
 
+    cartProduct: {
+      amountWidget: '.widget-amount',
+      price: '.cart__product-price',
+      edit: '.cart__product-edit',
+      remove: '.cart__product-remove',
```

Comment:
Nie masz elementów z takimi klasami w pliku html, dlatego przy próbie usunięcia produktu z koszyka nic się nie dzieje. Zamiast nazwy klas możesz tutaj użyć wartości atrybutu `href`:
```suggestion
      edit: '[href="#edit"]',
      remove: '[href="#remove"]',
```
Jeśli takie wartości przekażesz do metody `querySelector`, to złapiesz element, który posiada odpowiednio `href="#edit"` i  `href="#remove"`
Zauważ, że w Twoim pliku html masz takie elementy w koszyku:
```html
<li><a href="#edit">Edit<i class="fas fa-pencil-alt"></i></a></li>
<li><a href="#remove">Remove<i class="fas fa-trash-alt"></i></a></li>
```